bodhi-client/bodhi-client.spec

```diff
@@ -1,6 +1,7 @@
 # Created by pyp2rpm-3.3.7
 %global pypi_name bodhi-client
 %global pypi_version 5.7.5
+%global _pyproject_record %{_builddir}/%{pypi_name}-pyproject-record
```

Comment:
JFYI Trying to solve this in the pyproject macros themselves: https://src.fedoraproject.org/rpms/pyproject-rpm-macros/pull-request/260

Reply:
Fixed in:

 - [pyproject-rpm-macros-1.0.1-1.fc37](https://bodhi.fedoraproject.org/updates/FEDORA-2022-442e382b65)
 - [pyproject-rpm-macros-1.0.1-1.fc36](https://bodhi.fedoraproject.org/updates/FEDORA-2022-ba127f48b6)
 - [pyproject-rpm-macros-1.0.1-1.fc35](https://bodhi.fedoraproject.org/updates/FEDORA-2022-134c648316)
 - [pyproject-rpm-macros-1.0.1-1.fc34](https://bodhi.fedoraproject.org/updates/FEDORA-2022-9153b6e145)
 - [pyproject-rpm-macros-1.0.1-1.el9](https://gitlab.com/redhat/centos-stream/rpms/pyproject-rpm-macros/-/merge_requests/10)